src/jit/emit.cpp

```diff
@@ -51,7 +51,7 @@ int emitLocation::GetInsNum() const
     return emitGetInsNumFromCodePos(codePos);
 }
 
-#ifdef _TARGET_AMD64_
+#if defined(_TARGET_AMD64_) || (defined(_TARGET_ARM_) && defined(_TARGET_UNIX_))
```

Comment:
Does this need to be under ifdef at all?

Reply:
I don't think so, but I've only tested it on unix/arm

Reply:
I removed these ifdefs, the code seems to build ok.